src/store/playbackManager.ts

```diff
@@ -319,7 +319,10 @@ export const mutations: MutationTree<PlaybackManagerState> = {
     state.status = PlaybackStatus.Paused;
   },
   STOP_PLAYBACK(state: PlaybackManagerState) {
+    const volume = state.currentVolume;
+
     Object.assign(state, defaultState());
+    state.currentVolume = volume;
```

Comment:
I'm sure you have tested this :P, but just in case:

Are you sure volume isn't a reference to the state object? So as it's assigned by Object.assign, ``volume`` becomes the default value instead of the old one (which is the one we want to persist). So maybe we need to use Object.assign as well for volume.

Reply:
`volume` is assigned to the `currentVolume` value. It's only when passing around objects that you get a reference. And yes it was tested :p